.github/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+### Background
+
+(Provide a link to any relevant issues AND provide a TLDR of the issue in this pull request)
+
+### Solution
+
+(Describe your solution, why this approach was chosen, and what the alternatives/impacts may be)
+
+### 🎩
+
+- ...
+
+### Checklist
+
+- [ ] It is safe to simply rollback this change
```

Comment:
IMO this does not make sense for a library

Reply:
True, i'll remove it